### PR TITLE
Translate release posts (zh_tw)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

File: zh_tw/news/_posts/2018-10-17-ruby-2-3-8-released.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+---
+
+Ruby 2.3.8 已發佈。此次發佈修復了數個安全性缺陷，請查看下述主題以獲取詳情。
+
+* [CVE-2018-16396: 特定命令下受污染標記未能如實展開到 Array#pack 和 String#unpack 結果中](/zh_cn/news/2018/10/17/not-propagated-taint-flag-in-some-formats-of-pack-cve-2018-16396/)
+* [CVE-2018-16395: OpenSSL::X509::Name 相等檢查未正常工作](/zh_cn/news/2018/10/17/openssl-x509-name-equality-check-does-not-work-correctly-cve-2018-16395/)
+
+此次更新亦包含一個非安全性修復以支援 Windows 10 2018 年 10 月版本上的 Visual Studio。
+
+Ruby 2.3 處於安全維護期，并將於 2019 年 3 月底停止維護。此後 Ruby 2.3 的全部維護將終止。我們建議您開始規劃遷移到更新版本的 Ruby，例如 Ruby 2.5 或 2.4。
```
Comment (bugtender) [inline zh_tw/news/_posts/2018-10-17-ruby-2-3-8-released.md]: `并將` should be `並將`

File: zh_tw/news/_posts/2018-10-17-not-propagated-taint-flag-in-some-formats-of-pack-cve-2018-16396.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+lang: zh_tw
+---
+
+使用 `Array#pack` 和 `String#unpack` 處理某些特定格式時，原資料的受污染標記未能展開到返回的字串或陣列中。此缺陷已被分配 CVE 編號 [CVE-2018-16396](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-16396)。
+
+## 細節
+
+`Array#pack` 方法將其接收的內容轉換成某一特定格式。當參數包含一些受污染的物件，返回的字串也應包含這些污染。`Array#unpack` 方法將參數展開成一個陣列，也應該將受污染的標記轉換到返回的陣列中。然而，當命令含有 `B`、`b`、`H` 和 `h` 時，污染標記並不會被展開。所以，如果你的腳本使用 `Array#pack` 和/或 `String#unpack` 來處理了不可靠的輸入後，嘗試檢查這些受污染標記時，檢查結果可能出錯。
+
+所有用戶皆因儘快升級。
```
Comment (bugtender) [inline zh_tw/news/_posts/2018-10-17-not-propagated-taint-flag-in-some-formats-of-pack-cve-2018-16396.md]: Maybe `所有受到影響的用戶請儘快升級。` will be more clear.

File: zh_tw/news/_posts/2018-10-17-ruby-2-4-5-released.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      SIZE:   15694028 bytes
+      SHA1:   0bded7dfabfce5615162a1acd1341966a63e40f4
+      SHA256: 33694b03ac178cf96aa728b74de7b0bc5d848fcfabc64a7c74ea093198142601
+      SHA512: 9baec2b33604d5b2180b83bef9b39349d3f73b2eb5a7d44410572d893f76e6a0e1c39c572e6084a4e2466ca148ff2f377b5458144fe9b859a80cdf7b41fd5d72
+
+## 發佈記
+
+感謝所有幫助發佈這一版本的人。
+
+Ruby 2.4 的維護，包括這個版本在內，是基於 Ruby 協會的「Ruby 穩定版本協議」進行維護的的。
```
Comment (bugtender) [inline zh_tw/news/_posts/2018-10-17-ruby-2-4-5-released.md]: duplicate `的`

Comment (bugtender) [inline zh_tw/news/_posts/2018-10-17-ruby-2-4-5-released.md]: duplicate `的`

**Comment by dsh0416** [comment]
@bugtender Fixed. Thank you for your review.

**Comment by bugtender** [comment]
👍

**Pull request merged**